### PR TITLE
DOC-2056: Added TINY-9892 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2056

Changes:
* Added TINY-9892 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -49,6 +49,21 @@ This resulted in an inconsistent end-user experience.
...
+
+=== The insert template button and menu item was not disabled when the selection was in a non-editable context.
+//#TINY-9892
+
+In previous versions of {productname}, the insert template button and menu item remained enabled even in `contenteditable="false"` contexts.
+
+As a consequence, no action occured when attempting to insert a template into a `contenteditable="false"` root.
+
+In {productname} 6.5, the button and toolbar item will now be disabled when the selection is in a `contenteditable="false"` context.
+
```
Comment (spocke) [inline modules/ROOT/pages/6.5-release-notes.adoc:66]:
```suggestion
=== The insert footnotes button and menu item was not disabled when the selection was in a non-editable context.
//#TINY-9892

In previous versions of {productname}, the insert footnotes button and menu item remained enabled even in `contenteditable="false"` contexts.

As a consequence, no action occured when attempting to insert a footnote into a `contenteditable="false"` root.

In {productname} 6.5, the button and toolbar item will now be disabled when the selection is in a `contenteditable="false"` context.

```

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -49,6 +49,21 @@ This resulted in an inconsistent end-user experience.
 
 For information on the **Merge Tags** plugin, see: xref:mergetags.adoc[Merge Tags].
 
+=== Footnotes 1.0.1
+
+The {productname} 6.5 release includes an accompanying release of the **Footnotes** premium plugin.
+
+**Footnotes** 1.0.1 includes one fix:
```
Comment (spocke) [inline modules/ROOT/pages/6.5-release-notes.adoc:56]: Should this be here the ticket is for templates the ticket had the wrong component.

Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:56]:
```suggestion
=== Advanced Templates 1.2.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Templates** premium plugin.

**Advanced Templates** 1.2.0 includes one fix:
```

Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc]: @spocke Updated, thanks for the heads up.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -49,6 +49,21 @@ This resulted in an inconsistent end-user experience.
...
+**Advanced Templates** 1.2.0 includes one fix:
+
+=== The insert template button and menu item was not disabled when the selection was in a non-editable context.
+//#TINY-9892
+
+In previous versions of {productname}, the insert template button and menu item remained enabled even in `contenteditable="false"` contexts.
+
+As a consequence, no action occured when attempting to insert a template into a `contenteditable="false"` root.
+
+In {productname} 6.5, the button and toolbar item will now be disabled when the selection is in a `contenteditable="false"` context.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:65]:
```suggestion
=== The *Insert → Template…* menu item and equivalent toolbar button was not disabled when the selection was inside an element with a `contenteditable="false"` attribute set
//#TINY-9892


In previous versions of {productname}, the *Insert → Template…* menu item, and equivalent toolbar button remained active when the user’s selection or insertion point was inside an element with a `contenteditable="false"` attribute set.

As a consequence, the *Advanced Template* toolbar button and menu item appeared enabled, but clicking on them had no effect.

no action occured when attempting to insert a template into a `contenteditable="false"` root.

A fix was implemented in {productname} 6.5. {productname} now checks if the user’s current selection is within an element with a `contenteditable="false"` attribute set. If it is, the *Advanced Template* toolbar button and menu item are disabled, as expected.

For information on the **Advanced Templates** plugin, see: xref:advanced-templates.adoc[Advanced Templates].
```

1. spelling out _element with a `contenteditable="false"` attribute_ in full for correctness’s sake.
2. copy edits to UX and UI item spelling and presentation to exactly match presentation in the editor.
3. Other suggested edits.
4. Addition of the _For more information_ line containing an appropriate xref.

Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc:65]: @brian-forte Did you intend on leaving this in here?
`no action occured when attempting to insert a template into a `contenteditable="false"` root.`

Comment (brian-forte) [reply modules/ROOT/pages/6.5-release-notes.adoc:65]: > Did you intend on leaving this in here?

Good catch. I did not, and I’ve now removed it.

**Review by brian-forte — APPROVED** [review]

**Pull request merged**